src/entities/raw-data-providers/cardano-explorer-api/cardano-explorer-api.js

```diff
@@ -109,8 +110,8 @@ class CardanoExplorerApi implements RawDataProvider {
       isEBB: false,
       prevHash: blockData.previousBlockId && blockData.previousBlockId.toString('hex'),
       time: blockData.createdAt,
-      lead: blockData.createdBy,
-      slotLeaderPk: blockData.leaderHash && blockData.leaderHash.toString('hex'),
+      lead: issuerPk,
+      slotLeaderPk: Buffer.from(issuerPk, 'base64').toString('hex'),
```

Comment:
This doesn't look right.... `issuerPk` is generated from `toString('hex')` so why would we then try and parse it as base64?

Reply:
Additionally won't this fail if `blockData.leaderHash` is null?